From 653dd3f53610840baeba350c6520b0748305dd83 Mon Sep 17 00:00:00 2001
From: Nick Liu <nick_liu@avalue.com.tw>
Date: Wed, 1 Jun 2016 16:59:29 +0800
Subject: [PATCH 1/1] Modify Makefile

modified:   Makefile
---
 Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index e7354d6..884f4e2 100644
--- a/Makefile
+++ b/Makefile
@@ -18,9 +18,7 @@ kernel.bin: cleankernel
 	mkdir -p out
 	install kernel/arch/arm/boot/zImage out/zImage
 	install kernel/arch/arm/boot/dts/imx6q-rev-sa01-pfuze100.dtb out/imx6q-rev-sa01-pfuze100.dtb
-	#install kernel/arch/arm/boot/dts/imx6q-rev-sa01-wm8326.dtb out/imx6q-rev-sa01-wm8326.dtb
-	#install kernel/arch/arm/boot/dts/imx6solo-rev-sa01-pfuze100.dtb out/imx6solo-rev-sa01-pfuze100.dtb
-	#install kernel/arch/arm/boot/dts/imx6solo-rev-sa01-wm8326.dtb out/imx6solo-rev-sa01-wm8326.dtb
+	install kernel/arch/arm/boot/dts/imx6solo-rev-sa01-pfuze100.dtb out/imx6solo-rev-sa01-pfuze100.dtb
 
 ##########
 # U-Boot #
-- 
1.9.1

